### ISSUE TITLE
URL bar color contrast on dark theme

### ISSUE THREAD

**Issue by Hill-98** [zen-browser/desktop#2844]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

The color contrast of Zen Browser’s URL bar is a bit too low in dark mode, causing some accessibility issues and making it look like there’s no URL bar.

![图片](https://github.com/user-attachments/assets/e4e74358-890a-4ac9-a18f-fa915fc3e211)
![图片](https://github.com/user-attachments/assets/63b63d48-228f-4c9c-9a95-a43e19a94e15)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.18

### What platform are you seeing the problem on?

Windows
